ypathgen/pathgen.go

```diff
@@ -616,9 +619,11 @@ func mustTemplate(name, src string) *template.Template {
 // schema field names of that directory entry (i.e. the same keys as the
 // "Fields" map of the Directory entry). Since ygen provides a *MappedType for
 // every leaf node only, leafTypeMap's value is nil for non-leaf nodes.
+// packageName, splitByModule, and trimOCPackage are used to generate the name of
+// Go package for each output node.
```

Comment:
are used to determine the generated Go package name for the generated PathStructs.

Reply:
done